server/templates/partials/bottom/cookie-consent.html

```diff
@@ -8,7 +8,7 @@
 	}}
 	<h2 class="cookie-banner__head" id="cookie-banner-aria-label">Cookies on FT Sites</h2>
 	<div class="cookie-banner__inner" data-o-banner-inner="" data-o-banner-inner="">
-		<div id="cookie-banner-aria-description">We use <a href="http://help.ft.com/help/legal-privacy/cookies/" class="cookie-banner__link cookie-banner__link--external" data-o-banner-action data-o-banner-action-type="view-policy" target="_blank" rel="noopener">cookies</a> for a number of reasons, such as keeping FT Sites reliable and secure, personalising content and ads, providing social media features and to analyse how our Sites are used.</div>
+		<div id="cookie-banner-aria-description">We use <a href="http://help.ft.com/help/legal-privacy/cookies/" class="cookie-banner__link cookie-banner__link--external" data-o-banner-action data-o-banner-action-type="view-policy" target="_blank" rel="noopener">cookies <span class="o-normalise-visually-hidden">opens in new window</span></a> for a number of reasons, such as keeping FT Sites reliable and secure, personalising content and ads, providing social media features and to analyse how our Sites are used.</div>
```

Comment:
I think you should keep the `opens in new window` inside parenthesis cause I would expect a screen reader to do a pause after reading `cookies`. Maybe you can try it with VoiceOver in your machine and see the difference with or without.

I also believe the space should stay inside the span otherwise it will be visually rendered and it will sum with the space that is after the closure of the link, so you will have 2 spaces between `cookie` and `for a number of reasons`

```suggestion
		<div id="cookie-banner-aria-description">We use <a href="http://help.ft.com/help/legal-privacy/cookies/" class="cookie-banner__link cookie-banner__link--external" data-o-banner-action data-o-banner-action-type="view-policy" target="_blank" rel="noopener">cookies<span class="o-normalise-visually-hidden"> (opens in new window)</span></a> for a number of reasons, such as keeping FT Sites reliable and secure, personalising content and ads, providing social media features and to analyse how our Sites are used.</div>
```

Reply:
Thanks, good suggestions!